Pipfile

```diff
@@ -11,3 +11,5 @@ python_version = ">=3.5"
 [packages]
 python-dotenv = "*"
 requests = "*"
+pandas = "*"
+openpyxl = "*"
```

Comment:
we can remove openpyxl if it's unused

Reply:
This is needed for reading the .xlsx files

Reply:
Ah I see, okay nice :+1: